orm/src/test/java/org/hibernate/search/test/directoryProvider/MultipleSFTestCase.java

```diff
@@ -74,13 +75,13 @@ protected void buildSessionFactories(Class[] classes, String[] packages, String[
 		}
 	}
 
-	@Override
+	@Before
 	protected void setUp() throws Exception {
```

Comment:
You should probably make the method `public` just for consistency... Same for `tearDown`.